Automattic/jetpack
# Fix Protect UI footer plan check

#### Changes proposed in this Pull Request:
* At the moment, a section in the footer of the Protect plugin UI displays a message prompting the user to upgrade to the Jetpack Security bundle when that plan is identified as registered to the site. When the plan is not found, the message that is displayed appears to be what was intended for a site that has already been upgraded. In order to be effective, this logic needs to be reversed.

#### Jetpack product discussion


* Asana task: [1202346861908284](https://app.asana.com/0/1201069996155224/1202346861908284)

#### Does this pull request change what data or activity we track or use?


* No

#### Testing instructions:






* Checkout this branch
* Fire up your Jurassic Tube instance
* Ensure that the site does not currently have a Jetpack Security upgrade
* Activate the Jetpack Protect plugin
* Visit the Protect plugin page and view the footer section
* The following prompt should display:
<img width="305" alt="Screen Shot 2022-06-06 at 10 31 47" src="https://user-images.githubusercontent.com/43220201/172258961-c63a7df9-8330-4ddc-8ef1-11f1ea7e56fe.png">

* Apply a Security bundle to the site
* Visit the Protect plugin page and view the footer section again
* The following message should now display:
<img width="312" alt="Screen Shot 2022-06-06 at 10 30 39" src="https://user-images.githubusercontent.com/43220201/172258999-29920ba6-b0d0-4c8a-9028-b1476d5d6cb3.png">
